ci-operator/step-registry/cert-manager/custom-ingress-cert/cert-manager-custom-ingress-cert-commands.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
+    echo "proxy: ${SHARED_DIR}/proxy-conf.sh"
+fi
+
+CLUSTERISSUER_NAME=cluster-certs-clusterissuer
+if [[ ! "$(oc get --no-headers clusterissuer $CLUSTERISSUER_NAME)" =~ True ]]; then
+    echo "The prerequsite clusterissuer $CLUSTERISSUER_NAME is not ready. Please ensure the cert-manager-clusterissuer ref is executed first."
+    exit 1
+fi
+
+TMP_DIR=/tmp/cert-manager-ingress-commands-tmp-dir
+mkdir -p $TMP_DIR
+cd $TMP_DIR
+
+INGRESS_DOMAIN=$(oc get ingress.config cluster -o jsonpath='{.spec.domain}')
+CERT_NAME=custom-ingress-cert
+oc create -f - << EOF
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: $CERT_NAME
+  namespace: openshift-ingress
+spec:
+  commonName: "*.$INGRESS_DOMAIN"
+  dnsNames:
+  - "*.$INGRESS_DOMAIN"
+  usages:
+  - server auth
+  issuerRef:
+    kind: ClusterIssuer
+    name: $CLUSTERISSUER_NAME
+  secretName: cert-manager-managed-ingress-cert-tls
+# privateKey:
+#   rotationPolicy: Always # Venafi need this
+  duration: 2h
+  renewBefore: 1h30m
```

Comment:
Same as above in `custom-apiserver-cert-commands.sh`

Reply:
Same as above. If worry others in future won't understand its purpose, I can explain with comment.

Reply:
@lunarwhite , addressed your comments. Please take another look. If no more issue, please add /lgtm and @ lxia / kuiwang to review/merge. If having more issue, please let me know.